test/unit_tests/braket/default_simulator/openqasm/test_interpreter.py

```diff
@@ -1073,15 +1082,17 @@ def test_gphase():
     assert np.allclose(simulation.state_vector, [-1 / np.sqrt(2), 0, 0, 1 / np.sqrt(2)])
 
 
-def test_no_neg_ctrl_phase():
+def test_neg_ctrl_phase():
     qasm = """
-    gate bad_phase a {
-        negctrl @ gphase(π/2);
-    }
+    qubit q;
+    h q;
+    negctrl @ gphase(π) q;
+    h q;
```

Comment:
would this test be simpler to grok if the circuit was just `x q; negctrl @ gphase(π) q;`? I think this test would be extra compelling if we could also assert that it's equal to an equivalent circuit that uses x and ctrl @ gphase

Reply:
maybe h is better since it's more convincing we're not just getting lucky with the transformation since we're in a basis state